test/e2e/tests/confirmations/signatures/permit.spec.ts

```diff
@@ -14,12 +14,8 @@ import {
 import { Ganache } from '../../../seeder/ganache';
 import { Driver } from '../../../webdriver/driver';
 
-describe('Confirmation Signature - Permit', function (this: Suite) {
-  if (!process.env.ENABLE_CONFIRMATION_REDESIGN) {
-    return;
-  }
-
-  it('initiates and confirms', async function () {
+describe('Confirmation Signature - Permit @no-mmi', function (this: Suite) {
+  it('initiates and confirms and emits the correct events', async function () {
```

Comment:
+1 extra line to resolve merge conflict - LGTM  ([compare line](https://github.com/MetaMask/metamask-extension/pull/25520/files#diff-9b7dc3b211ccc46387c285548ff2177a7f4f499a74715bba074069f0c7f82222L26))